bionty/core/_bionty.py

```diff
@@ -115,3 +122,55 @@ def _prepare_bionty_df(model: type[Record], bionty_df: pd.DataFrame):
     # rename definition to description for the bionty registry in db
     bionty_df.rename(columns={"definition": "description"}, inplace=True)
     return bionty_df
+
+
+def register_source_in_bionty_assets(
+    filepath: Path,
+    source: Record,
+    is_dataframe: bool = True,
+    update: bool = False,
+):
+    """Register a new source in the laminlabs/bionty-assets instance.
+
+    Args:
+        filepath (Path | str): Path to the source file.
```

Comment:
Kind request to remove the autogenerated type annotations :)

Reply:
oOPS!

Reply:
The manual type annotations!